pkg/controllers/configobservation/configobservercontroller/observe_config_controller.go

```diff
@@ -89,6 +89,7 @@ func NewConfigObserver(
 
 			APIServerLister_:     configInformer.Config().V1().APIServers().Lister(),
 			ConsoleLister:        configInformer.Config().V1().Consoles().Lister(),
```

Comment:
If the console type does not exist, the lister presence will fill our logs with error messages in a loop. We can't have that.